packages/devtools_server/lib/src/service_registrations.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
go ahead and just move more of this to shared. we can actually use the icons.

Reply:
I'm going to do this post this PR (it's a pain to update shared then commit, publish, and update everything in this PR).  I want to move some more common code in devtools_app and devtools_server to shared (how HeapSpace, HeapSamples, etc. are constructed is same code).